src/sentry/integrations/helper.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import absolute_import, print_function
+
+__all__ = ['IntegrationSetupHelper']
+
+import logging
+
+from django.contrib import messages
+from django.http import HttpResponseRedirect
+
+from sentry.models import Integration
+from sentry.utils.hashlib import md5_text
+from sentry.utils.http import absolute_uri
+from sentry.web.helpers import render_to_response
+
+from . import manager
+
+SESSION_KEY = 'integration.setup'
+
+logger = logging.getLogger('sentry.integrations')
+
+
+class IntegrationSetupHelper(object):
```

Comment:
core of this will probably turn into something like 'PipelineHelper' and then AuthHelper can just extend it